apps/nextjs/src/app/(dashboard)/dashboard/(body)/layout.tsx

```
@@ -1,33 +1,17 @@
 }
 
-export default function SharedLayout({
-  children,
-  // left,
-  // center,
-  // right,
-}: LayoutProps) {
+export default function SharedLayout({ children }: LayoutProps) {
   return (
     <section
       role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)